_docs/_latest/develop/dev/inventory.md

```diff
@@ -6,35 +6,37 @@ order: 102
 # {{ page.title }}
 
 This page describes how to update Forseti Inventory to collect and store new
-data types, as well as importing it into data model.
+data types, and import it into a data model.
 
 ---
 
-The main steps to add a new type of Inventory data are:
-1. Check API client exists
-1. Create iterator to retrieve the data with the API client
+To add a new type of Inventory data, you'll complete the following tasks:
+
+1. Check that the API client exists
+1. Create an iterator to retrieve the data with the API client
 1. Add the iterator to Inventory factories to make it available for crawling
-1. Add the Inventory data to data model
+1. Add the Inventory data to a data model
 
-To see how these steps actually works, we will do a code-walk of
+The following guide demonstrates these steps as a walkthrough of 
 [PR #883](https://github.com/GoogleCloudPlatform/forseti-security/pull/883),
-which adds Compute Image data to Inventory and data model.
-
-## Check API clients exists
+which adds Compute Engine Image data to Inventory and a data model.
 
-Look in [_supported_apis.py](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/google/cloud/forseti/common/gcp_api/_supported_apis.py), if the API client to retrieve the data already exists.
-If the API client is not found, you will have to add it. See [cloud_sql.py](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/google/cloud/forseti/common/gcp_api/cloudsql.py)
-for a small and self-contained example.
+## Step 1: Check the API client
 
-## Create iterator to retrieve the data with the API client
+To check if the API client to retrieve the data already exists, look in [`_supported_apis.py`](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/google/cloud/forseti/common/gcp_api/_supported_apis.py).
```

Comment:
@blueandgold What do I look for in _supported_apis.py?

Reply:
Added detail to look at the SUPPORTED_APIS map.